src/lib/stores/modals.svelte.ts

```diff
@@ -1,10 +1,10 @@
-import { Map } from "svelte/reactivity";
+import {  SvelteMap } from "svelte/reactivity";
 import { set_all_to_value } from "$functions/map/set_all_to_value";
 
 const modals = ["search"] as const;
 type IModals = (typeof modals)[number];
 
-var state_map = $state(new Map<IModals, boolean>(modals.map((item) => [item, false])));
+var state_map = $state(new SvelteMap<IModals, boolean>(modals.map((item) => [item, false])));
```

Comment:
This should be a `let`

Reply:
Its same as previous, even when I cloned

Reply:
> Its same as previous, even when I cloned

Ah maybe i overlooked it. But please change it to `let`?

Reply:
Changes made...